tests/ha/check_after_reboot.pm

```diff
@@ -39,6 +39,12 @@ sub run {
     # in that case
     select_console 'root-console' if (get_var('HDDVERSION'));
 
+    script_run 'hostname';
+    script_run "hostnamectl status";
++   assert_script_run "cat /etc/hostname";
+    save_screenshot;
+
+
```

Comment:
is this for testing only? At least it sounds unrelated to the commit message.

Reply:
Exactly, it's a remnant of my previous PR, I removed it. Thanks for pointing it.